### PR TITLE
Additional improvements

### PR DESCRIPTION
**Pull request by OlegSternbergIBM**

### CODE REVIEW

File: pkg/controller/bundles/bundles_syncer.go [hunk 1]
```diff
@@ -87,6 +95,26 @@ func (syncer *LeafHubBundlesSpecSync) Start(stopChannel <-chan struct{}) error {
 	return nil
 }
 
+func readK8sPoolSizeEnvVar(log logr.Logger) int {
+	envK8sClientsPoolSize, found := os.LookupEnv(envVarK8sClientsPoolSize)
+	if !found {
+		log.Info("readK8sPoolSizeEnvVar", fmt.Sprintf("env variable %s not found", envVarK8sClientsPoolSize),
```
Comment (nirrozenbaum) [inline pkg/controller/bundles/bundles_syncer.go]: no need for function name, no need for double use of fmt.Sprintf.
```
log.Info(fmt.Sprintf("env variable %s not found, using default value %d", envVarK8sClientsPoolSize,
	defaultK8sClientsPoolSize))
```

Comment (nirrozenbaum) [inline pkg/controller/bundles/bundles_syncer.go:101]: no need for function name, no need for double use of fmt.Sprintf
```
log.Info(fmt.Sprintf("env variable %s not found, using default value %d", envVarK8sClientsPoolSize,
	defaultK8sClientsPoolSize))
```

File: pkg/controller/bundles/bundles_syncer.go [hunk 2]
```diff
@@ -87,6 +95,26 @@ func (syncer *LeafHubBundlesSpecSync) Start(stopChannel <-chan struct{}) error {
...
+			fmt.Sprintf("using default value %d", defaultK8sClientsPoolSize))
+
+		return defaultK8sClientsPoolSize
+	}
+
+	value, err := strconv.Atoi(envK8sClientsPoolSize)
+	if err != nil {
+		log.Info("readK8sPoolSizeEnvVar", fmt.Sprintf("failed to convert variable %s, value: %s",
```
Comment (nirrozenbaum) [inline pkg/controller/bundles/bundles_syncer.go]: no need to log function name, no need for double use of fmt.Sprintf.
need to add another check for illegal value (added <1 check)
```
value, err := strconv.Atoi(envK8sClientsPoolSize)
if err != nil || value < 1 {
	log.Info(fmt.Sprintf("env var %s invalid value: %s, using default value %d", envVarK8sClientsPoolSize,
		envK8sClientsPoolSize, defaultK8sClientsPoolSize))
```

File: pkg/controller/bundles/bundles_syncer.go [hunk 3]
```diff
@@ -87,6 +95,26 @@ func (syncer *LeafHubBundlesSpecSync) Start(stopChannel <-chan struct{}) error {
 	return nil
 }
 
+func readK8sPoolSizeEnvVar(log logr.Logger) int {
+	envK8sClientsPoolSize, found := os.LookupEnv(envVarK8sClientsPoolSize)
```
Comment (nirrozenbaum) [inline pkg/controller/bundles/bundles_syncer.go]: can be called k8sClientsPoolSize (no need for the env prefix)

**Review by nirrozenbaum — COMMENTED** [review]
@OlegSternbergIBM minor comments in readEnvVars func

File: pkg/controller/bundles/bundles_syncer.go (hunk 3 above)
Comment (OlegSternbergIBM) [reply pkg/controller/bundles/bundles_syncer.go]: done

File: pkg/controller/bundles/bundles_syncer.go (hunk 2 above)
Comment (OlegSternbergIBM) [reply pkg/controller/bundles/bundles_syncer.go]: done

**Review by nirrozenbaum — APPROVED** [review]

**Pull request merged**